documentation/Installation.md

```diff
@@ -165,16 +165,16 @@ For cluster machines, ensure the following requirements are met:
 
```

Comment:
I see rhel 7 still mentioned in several places
* Installation.md
* software_upgrade.yaml (and related files)
* packages.py, yum.py
* in the examples it seems rhel/centos 7 repost are still mentioned

I believe we need to remove those too.

Also, after we change `software_upgrade.yaml` strcuture, I believe we need to dev-test that third-party upgrade procedure still works for packages (e.g. try to upgrade containerd, check that patch is created and that it is applied)